ui/src/hooks/useFileManage.ts

```diff
@@ -500,8 +500,8 @@ const handleFileUpload = (
           const percent = (sentChunks / sliceChunks.length) * 100;
 
           console.log(
-            '%c DEBUG[ percent ]-79:',
-            'font-size:13px; background:#F0FFF0; color:#800000;',
+            '%c DEBUG[ percent ]:',
+            'font-size:13px; background: #1ab394; color:#fff;',
             percent
           );
 
```

Comment:
The above code does not have any obvious mistakes or issues. However, here is a brief suggestion to optimize the performance of loading files:

For uploading files via SFTP, increase chunk size (`CHUNK_SIZE`) from `<1k` MB up to `<3k` MB or even greater depending on network conditions.

This optimization would reduce unnecessary I/O operations that can lead to better resource utilization and potentially faster transfer times, especially for large file uploads.